.github/workflows/pr.yaml

```diff
@@ -175,6 +176,17 @@ jobs:
       script: "ci/test_python_scikit_learn_tests.sh"
       # One run for each dependencies config on amd64, breaking ties by highest Python version
       matrix_filter: map(select(.ARCH == "amd64")) | sort_by(.PY_VER) | unique_by(.DEPENDENCIES)
+  # XXX Only here to get some testing while developing, should be removed before merging
+  # Not newest or oldest but "intermediate" dependencies
+  conda-python-scikit-learn-accel-tests-intermediate:
```

Comment:
Instead of adding a new build, can we instead apply the `matrix_filter` to the existing sklearn test run? Then we'd have one job (which has 2 runs in PR CI testing 2 sklearn versions, and 4 runs in nightly CI testing 3 sklearn versions).

Or is that what you were suggesting in the comment above?

Reply:
The comment is trying to remind me to remove this before merging, the real change should be in `test.yaml` which is where the nightlies are. But that doesn't run as part of a PR, so hard to test.

---

I'm not sure (aka my `jq` ninja foo is too low) if we can create a `matrix_filter` that adds an entry to the thing it is filtering. If we could do that, we'd avoid having a additional job in the workflow. This would be nice. I'll see if I can cook up a `matrix_filter`.